src/cdk/public_api.ts

```diff
@@ -6,4 +6,10 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+export * from './a11y/index';
+export * from './bidi/index';
 export * from './coercion/index';
+export * from './platform/index';
+export * from './portal/index';
+export * from './rxjs/index';
+export * from './keyboard/keycodes';
```

Comment:
For the CDK I wanted the convention to always have an `index.ts` file in the given piece of the CDK.

The index file of the "CDK piece" then always specifies what should be exported.

For the keycodes it might not be necessary, but for the sake of consistency I think it would make sense to do that as well (otherwise it will end up as in core.ts)